build/templates/_converters.py.mako

```diff
@@ -158,6 +158,98 @@ def convert_repeated_capabilities_without_prefix(repeated_capability):
     return ','.join(convert_repeated_capabilities(repeated_capability, ''))
 
 
+def convert_single_group_repeated_capabilities(repeated_capability):
+    '''Convert a repeated capabilities string, with at most one instrument prefix (that ends with '/') and without any comma, to a list
+
+    Examples:
+        - '0' --> ['0']
+        - '0-2' --> ['0', '1', '2']
+        - '0:2' --> ['0', '1', '2']
+        - 'Dev1/0' --> ['Dev1/0']
+        - 'Dev1/0-2' --> ['Dev1/0', 'Dev1/1', 'Dev1/2']
+        - 'Dev1/0:2' --> ['Dev1/0', 'Dev1/1', 'Dev1/2']
+
+    Args:
+        repeated_capability (str) - Supported types (with an optional instrument prefix):
+            - range (using '-' or ':')
+            - single item
+
+    Returns:
+        rep_cap_list (list of str) - list of each expanded repeated capability item
+    '''
+    if '/' in repeated_capability:
+        split_index = repeated_capability.find('/') + 1
+        instrument_prefix = repeated_capability[:split_index]
+        return convert_repeated_capabilities(repeated_capability[split_index:], instrument_prefix)
+    return convert_repeated_capabilities(repeated_capability)
+
+
+def convert_channels_repeated_capabilities(
+    channels_repeated_capability,
+    session_channel_names
+):
+    '''Convert a channels repeated capabilities string, possibly with no or multiple prefixes (each ends with '/'), to a list
+
+    Examples:
+        - convert_channels_repeated_capabilities('1', ['0', '1', '2', '3']) --> ['1']
+        - convert_channels_repeated_capabilities('0-2', ['0', '1', '2', '3']) --> ['0', '1', '2']
```

Comment:
Suggest having less examples here, just enough to figure out what the function does.
The unit tests should be comprehensive of course.

Reply:
I have removed some unnecessary examples for the new converter function `expand_channel_string()`.